### PR TITLE
Update Versions - Cats RC2

### PR DESCRIPTION
**Pull request by rsoeldner**
We need to wait:

- [x] http4s version 0.18.0-M7
- [x] circe version 0.9.0-M3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ internals shift as we find better/more performant abstractions.
 
 We will guarantee compatibility between minor versions (i.e 0.0.1 => 0.0.2) but not major versions (0.0.1 => 0.1.0)
 
-0.0.1-M6 is here for scala 2.12+ and Cats 1.0.0-RC1!
+0.0.1-M7 is here for scala 2.12+ and Cats 1.0.0-RC2!
```
Comment (hrhino) [inline README.md]: Are we releasing immediately after the upgrade? Otherwise this is misleading.

**Comment by hrhino** [comment]
LGTM otherwise

File: README.md (hunk 1 above)
Comment (jmcardon) [reply README.md]: let me merge this one. I can make sure we release after update iff I merge this

Comment (rsoeldner) [reply README.md]: we are waiting for http4s :+1:

**Comment by rsoeldner** [comment]
@jmcardon After travis gives a green smile, we are ready to release :tada:

**Comment by jmcardon** [comment]
Just as an update, I want to get #88 and #90 in there before we merge if possible.

Both of those are basically QOL improvements on the tsec http4s api which make using it much nicer

**Pull request merged**